app/views/publishers/feedback_prompt_mailer/prompt_for_feedback.text.erb

```diff
@@ -8,9 +8,9 @@ We’re constantly working to improve <%= t('app.title') %>. You can help us do
   <%="* #{vacancy.job_title}" %>
 <% end %>
 
-It should only take a minute or 2 of your time. Just <%= notify_link(new_publisher_session_url, "sign into your Teaching Vacancies account") %> and select the ‘Jobs awaiting feedback’ tab.
+It should only take a minute or 2 of your time. Just <%= publisher_sign_in_link %> and select the ‘Jobs awaiting feedback’ tab.
```

Comment:
@cpjmcquillan are we accidentally losing the link text from the email here (sign into your Teaching Vacancies account etc.)?